mtt/mastodon_to_twitter.py

```diff
@@ -229,6 +229,6 @@ def on_update(self, toot):
 
         # Compatibility with multiple versions of Mastodon.py
         try:
-            self.mastodon_api.stream_user(TootsListener(self), async=False)
```

Comment:
api change in https://github.com/halcy/Mastodon.py/commit/2afc50c803177fc282fac1a19fcd40e175d55f2d#diff-4a4f11c7204faf2aad3ce8e1188b86d2